src/components/Activities/index.jsx

```diff
@@ -11,7 +13,7 @@ export default class Activities extends React.Component {
       title_primary: null,
       title_complementary: null,
       description: null,
-      loading: true,
+      loading: false,
```

Comment:
why you define it as false .. 
loading didn't appear because it's won't be true in any state 
i think you need to define it as true